src/snowflake/snowpark/modin/plugin/docstrings/window.py

```diff
@@ -499,7 +499,36 @@ def max():
         pass
 
     def corr():
-        pass
+        """
+        Calculate the rolling correlation.
+
+        Parameters
+        ----------
+        other : Series or DataFrame, optional
+            If not supplied then will default to self and produce pairwise output.
+        pairwise : bool, default None
+            If False then only matching columns between self and other will be used and the output will be a DataFrame. If True then all pairwise combinations will be calculated and the output will be a MultiIndexed DataFrame in the case of DataFrame inputs. In the case of missing elements, only complete pairwise observations will be used.
```

Comment:
Nit: Can we split this out into multiple lines and clean up the formatting a bit?